project1/src/components/Article/ArticleInfo.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import moment from "moment";
+
+interface ArticleInfoProps {
+  image: string;
+  username: string;
+  createdAt: string;
+}
+
+const ArticleInfo: React.FC<ArticleInfoProps> = props => {
+  return (
+    <>
+      {/* <Grid item>
+        <a>
+          <img
+            src={props.image}
+            className="articleImage"
+            alt={props.username}
+          ></img>
+        </a>
+      </Grid>
+      <Grid item>
+        <div>
+          <a>{props.username}</a>
+        </div>
+        <span>
+          {moment(new Date(Date.parse(props.createdAt))).format("MMMM D, YYYY")}
+        </span>
+      </Grid> */}
+
+      <Grid
+        item
+        container
+        direction="row"
+        alignItems="center"
+        xs={8}
+        spacing={2}
+      >
+        <Grid item>
+          <img
+            src={props.image}
+            className="articleImage"
+            alt={props.username}
+          ></img>
+        </Grid>
+        <Grid item>
+          <Grid>{props.username}</Grid> {/* Make this a Link */}
+          <Grid className="date">
+            {moment(new Date(Date.parse(props.createdAt))).format(
```

Comment:
do the calculations at the top of the function, and why do you need to create a date then a moment, you should be able to create moment from a string directly